From b009cca2026b615ef5386faa4c0230bc27c4161d Mon Sep 17 00:00:00 2001
From: Greg Hudson <ghudson@mit.edu>
Date: Thu, 12 Mar 2020 00:44:10 -0400
Subject: [PATCH] Fix typo in musl build fix

Commit cbdbc8d00d31344fafe00e0fdf984e04e631f7c4 checked for
__GLIBC__PREREQ instead of __GLIBC_PREREQ, thus accidentally reverting
the workaround introduced in commit
bf5953c549a6d279977df69ffe89b2ba51460eaf.  Fix the typo.

ticket: 8880
---
 src/util/support/plugins.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/util/support/plugins.c b/src/util/support/plugins.c
index 1644d16fd0..1ff10c354d 100644
--- a/src/util/support/plugins.c
+++ b/src/util/support/plugins.c
@@ -62,7 +62,7 @@
  * dlopen() with RTLD_NODELETE, we weren't going to unload the plugin objects
  * anyway.
  */
-#ifdef __GLIBC__PREREQ
+#ifdef __GLIBC_PREREQ
 #if ! __GLIBC_PREREQ(2, 25)
 #define dlclose(x)
 #endif
